mcstatus/server.py

```diff
@@ -57,16 +61,14 @@ def lookup(cls, address: str, timeout: float = 3) -> Self:
         :param str address: The address of the Minecraft server, like `example.com:19132`
         :param float timeout: The timeout in seconds before failing to connect.
         """
-        addr = Address.parse_address(address, default_port=None)
+        addr = Address.parse_address(address, default_port=cls.default_port)
         return cls(addr.host, addr.port, timeout=timeout)
 
 
 class JavaServer(MCServer):
     """Base class for a Minecraft Java Edition server."""
 
-    def __init__(self, host: str, port: int = 25565, timeout: float = 3):
-        """Override init to add a default port for java servers of 25565."""
-        super().__init__(host, port, timeout=timeout)
+    default_port: int = 25565
```

Comment:
The int type-hint here is redundant, same for setting this variable in the bedrock class.